OmiseSDK/CreditCardFormViewController.swift

```diff
@@ -767,14 +767,15 @@ extension CreditCardFormViewController {
         })
         
         confirmButton.titleLabel?.adjustsFontForContentSizeCategory = true
-        
-        let fieldsAccessibilityElements = ([
-            cardNumberTextField.accessibilityElements?.first ?? cardNumberTextField as Any,
-            cardNameTextField.accessibilityElements?.first ?? cardNameTextField as Any,
-            expiryDateTextField.expirationMonthAccessibilityElement as Any,
-            expiryDateTextField.expirationYearAccessibilityElement as Any,
-            secureCodeTextField.accessibilityElements?.first ?? secureCodeTextField as Any,
-            ]).compactMap({ $0 as? NSObjectProtocol })
+   
+//      Not used
+//        let fieldsAccessibilityElements = ([
+//            cardNumberTextField.accessibilityElements?.first ?? cardNumberTextField as Any,
+//            cardNameTextField.accessibilityElements?.first ?? cardNameTextField as Any,
+//            expiryDateTextField.expirationMonthAccessibilityElement as Any,
+//            expiryDateTextField.expirationYearAccessibilityElement as Any,
+//            secureCodeTextField.accessibilityElements?.first ?? secureCodeTextField as Any,
+//            ]).compactMap({ $0 as? NSObjectProtocol })
```

Comment:
You can remove this property instead of using the `comments`.

Reply:
Removed